kolibri/plugins/management/assets/src/vue/user-page/index.vue

```diff
@@ -220,6 +221,7 @@
     &:focus
       outline: none
       border-color: transparent
+      border-bottom: 2px solid $core-action-normal
```

Comment:
This seems like an inconsistent focus style.

I'll switch it to copy the learn/explore page's styling for now